frappe/email/doctype/notification/notification.py

```diff
@@ -624,10 +626,25 @@ def get_receiver_list(self, doc, context, field_on_user="mobile_no", recipient_e
 
 		return list(set(receiver_list))
 
-	def get_attachment(self, doc):
-		"""check print settings are attach the pdf"""
-		if not self.attach_print:
-			return None
+	def get_attachments(self, doc) -> list[dict]:
```

Comment:
```suggestion
	def get_attachment(self, doc) -> list[dict]:
```

While this name is better, let's not change it in order to maintain backwards compatibility.